x-pack/plugins/lens/public/editor_frame_service/editor_frame/editor_frame.test.tsx

```diff
@@ -37,16 +36,17 @@ import { fromExpression } from '@kbn/interpreter/common';
 import {
   createMockVisualization,
   createMockDatasource,
-  createExpressionRendererMock,
   DatasourceMock,
-} from '../mocks';
+  createExpressionRendererMock,
+} from '../../mocks';
 import { ReactExpressionRendererType } from 'src/plugins/expressions/public';
 import { DragDrop } from '../../drag_drop';
-import { FrameLayout } from './frame_layout';
 import { uiActionsPluginMock } from '../../../../../../src/plugins/ui_actions/public/mocks';
 import { chartPluginMock } from '../../../../../../src/plugins/charts/public/mocks';
 import { expressionsPluginMock } from '../../../../../../src/plugins/expressions/public/mocks';
 import { mockDataPlugin, mountWithProvider } from '../../mocks';
+import { setState, setToggleFullscreen } from '../../state_management';
+import { FrameLayout } from './frame_layout';
 
 function generateSuggestion(state = {}): DatasourceSuggestion {
   return {
```

Comment:
Can't comment there but in line 71 there's still an `onError` prop - seems to be a leftover